code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -667,12 +675,13 @@ var/global/list/robot_modules = list(
 	RG.add_reagent(/decl/reagent/polysomnine/beer2, 50)
 	emag.name = "Mickey Finn's Special Brew"
 
-/obj/item/robot_module/clerical/general
+/obj/item/robot_module/service/clerical
 	name = "clerical robot module"
 	channels = list(CHANNEL_SUPPLY = TRUE, CHANNEL_COMMAND = TRUE)
 	networks = list(NETWORK_MINE)
+	specialized_access_types = list(/datum/job/janitor) // Janitor is a nice general access without specifics
```

Comment:
directly define the general departmental accesses instead, if they are required.

Reply:
That will require a lot of remapping, might be too out of scope of the PR.

Reply:
`list(access_medical, access_engine, access_cargo, access_research, access_sec_doors)`
or 5, 10, 31, 47, 63 for respective numbers.

Reply:
This list accepts job types, not specific access defines. Reason is so that when job's access is updated we don't need to maintain these references. Making it what you suggest will add more maintenance and make lists very large.

Reply:
i think that it should support proper access datums, but i will not force you to implement this if you do not want to.

Reply:
> i think that it should support proper access datums, but i will not force you to implement this if you do not want to.

Maybe later, Chads is waiting for this PR to start their part of cyborg rework